docs/versions/unversioned/expokit/expokit.md

```diff
@@ -83,6 +83,39 @@ ExpoKit's release cycle follows the Expo SDK release cycle. When a new version o
 - Open up `ios/Podfile` in your project, and update the `ExpoKit` tag to point at the [release](https://github.com/expo/expo/releases) corresponding to your SDK version. Run `pod update` then `pod install`.
 - Open `ios/your-project/Supporting/EXSDKVersions.plist` in your project and change all the values to the new SDK version.
 
+If upgrading from SDK 31 or below, you'll need to refactor `AppDelegate` class as we moved its Expo-related code to separate class `EXStandaloneAppDelegate` owned by `ExpoKit` to simplify future upgrade process as much as possible. As of SDK 32, your `AppDelegate` class needs to subclass `EXStandaloneAppDelegate` and override its methods if you need to add any custom behavior to the delegate, but then remember to call the same method from the superclass (see `application:didFinishLaunchingWithOptions:` in `AppDelegate.m` below). Here are the basic `AppDelegate` files without any custom behavior:
+
+`AppDelegate.h`:
+```objc
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import <UIKit/UIKit.h>
+#import <ExpoKit/EXStandaloneAppDelegate.h>
+
+@interface AppDelegate : EXStandaloneAppDelegate <UIApplicationDelegate>
```

Comment:
We don't need this, but I guess it's a good practice since `AppDelegate` does implement `UIApplicationDelegate` even without extending `EXStandaloneAppDelegate`.